build.gradle

```diff
@@ -18,17 +18,38 @@ install {
 }
 
 repositories {
+    if (project.hasProperty('local')) {
+        mavenLocal()
+    }
     jcenter()
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+    maven { url 'http://repository.activeeon.com/content/groups/proactive/'}
 }
 
 task wrapper(type: Wrapper) {
     gradleVersion = '2.13' //version required
 }
 
+configurations {
+    provided
+}
+
+sourceSets {
+    main {
+        main.compileClasspath += configurations.provided
+        test.compileClasspath += configurations.provided
+        test.runtimeClasspath += configurations.provided
+    }
+}
+
 
 dependencies {
+    provided 'org.ow2.proactive:scheduler-api:+'
```

Comment:
Are you sure we don not want to use specific version?

Reply:
we do this for all script engines. I am not sure it's the best, but it's complex to do otherwise, and I don't have time.